### ISSUE TITLE
result of the pretrained model

### ISSUE THREAD

**Issue by jingweiz** [junyanz/CycleGAN#57]
Hey,
thanks for publishing the pretrained model, however, after I run the scripts that you provided to generate Paul Cézanne style images, here are the images that I get:
![screenshot from 2017-10-12 16 47 40](https://user-images.githubusercontent.com/9096283/31502368-26adbb24-af6d-11e7-89d2-57a60c1dfe72.png)
I don't know where went wrong cos I just follow directly your instructions here:
![screenshot from 2017-10-12 16 49 06](https://user-images.githubusercontent.com/9096283/31502415-4a24ea8c-af6d-11e7-8095-cb698d39248d.png)
Any help would be appreciated, thanks in advance!

**Comment by Quasimondo** [comment]
I suspect that there has been an update to torch or one of the required libraries that break something since I am suddenly seeing similar results with models I trained myself a few weeks ago (and which worked just fine)

**Comment by junyanz** [comment]
Interesting. I am currently traveling at ICCV. WIll look at the issue soon.

**Comment by junyanz** [comment]
The code still works for me with the old Torch version. Here is my version number with `luarocks show torch7`.
```bash
torch scm-1 - Torch7



License: 	BSD
Homepage: 	https://github.com/torch/torch7
Installed in: 	/data1/junyanz/torch/install

Modules:
	libtorch (/data1/junyanz/torch/install/lib/lua/5.1/libtorch.so)
	torch (/data1/junyanz/torch/install/share/lua/5.1/torch/init.lua)
	torch.CmdLine (/data1/junyanz/torch/install/share/lua/5.1/torch/CmdLine.lua)
	torch.FFI (/data1/junyanz/torch/install/share/lua/5.1/torch/FFI.lua)
	torch.File (/data1/junyanz/torch/install/share/lua/5.1/torch/File.lua)
	torch.Tensor (/data1/junyanz/torch/install/share/lua/5.1/torch/Tensor.lua)
	torch.TestSuite (/data1/junyanz/torch/install/share/lua/5.1/torch/TestSuite.lua)
	torch.Tester (/data1/junyanz/torch/install/share/lua/5.1/torch/Tester.lua)
	torch.paths (/data1/junyanz/torch/install/share/lua/5.1/torch/paths.lua)
	torch.test (/data1/junyanz/torch/install/share/lua/5.1/torch/test.lua)
	torch/README.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/README.md)
	torch/doc/cmdline.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/cmdline.md)
	torch/doc/diskfile.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/diskfile.md)
	torch/doc/file.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/file.md)
	torch/doc/gather.png (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/gather.png)
	torch/doc/index.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/index.md)
	torch/doc/maths.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/maths.md)
	torch/doc/memoryfile.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/memoryfile.md)
	torch/doc/pipefile.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/pipefile.md)
	torch/doc/random.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/random.md)
	torch/doc/serialization.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/serialization.md)
	torch/doc/storage.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/storage.md)
	torch/doc/tensor.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/tensor.md)
	torch/doc/tester.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/tester.md)
	torch/doc/timer.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/timer.md)
	torch/doc/utility.md (/data1/junyanz/torch/install/lib/lua/5.1/torch/doc/utility.md)
```

**Comment by junyanz** [comment]
I resolved the issue by
- removed the old torch and installed a new torch from the scratch following this official [instruction](http://torch.ch/docs/getting-started.html). 
Here are the results for the first 5 test images. 
The thing I spent a few hours but didn't work was trying to manually upgrade torch7/nn/etc. module.  I will recommend that you install a new Torch from the scratch.
![pretrained_model](https://user-images.githubusercontent.com/1924757/32215711-fe7e9b88-bddf-11e7-9825-fe310ff23657.png)
